interfaces/Portalicious/src/app/pages/project-registrations/components/change-status-dialog/change-status-dialog.component.ts

```diff
@@ -126,6 +130,26 @@ export class ChangeStatusDialogComponent
     return statusesWithSendMessageEnabled.includes(status);
   });
 
+  reasonValidationErrorMessage = computed(() => {
+    if (!this.reason() && this.reasonIsRequired()) {
+      return $localize`Please add a reason`;
```

Comment:
Rather than adding a new localization string:
```suggestion
      return $localize`:@@generic-required-field:This field is required.`;
```